docker.go

```diff
@@ -221,6 +222,9 @@ func commandBuild(build Build) *exec.Cmd {
 	if build.NoCache {
 		args = append(args, "--no-cache")
 	}
+	if build.ShmSize != "" {
+		args = append(args, "--shm-size=" + build.ShmSize)
```

Comment:
```diff
-		args = append(args, "--shm-size=" + build.ShmSize)
+		args = append(args, "--shm-size", build.ShmSize)
```

Reply:
Thanks for your review !

Pushed your fix